_docs/style/spacing.md

```diff
@@ -0,0 +1,290 @@
+
+## Introduction
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol-12 hxCol-xs-12 hxCol-sm-12 hxCol-md-12 hxCol-lg-12" %}
+
+The Helix spacing system was influenced by [Priyanka Godbole’s article](https://blog.prototypr.io/a-framework-for-creating-a-predictable-and-harmonious-spacing-system-8eee8aaf773c) in which she outlines steps for defining a system based on the **rule of three C’s**.
+
+- [**Containers**](#containers): A “square” inset of twenty pixels is applied to the top, right, bottom, and left sides of any container.
+- [**Content**](#content): A “stack” value is applied to the space below [headers](#headers) (eight pixels) and [leaf-nodes](#leaf-nodes) (four, eight, or sixteen pixels according to content type)
+- [**Components**](#components): “Inline” spacing values are appled to the space between components.  This rule uses eight pixels in most cases, and four pixels for close proximity association.
+
+This framework enabled us to define the spacing system using a limited set of values and rules that facilitate information hierarchy and adhere to accessibility guidelines.
+For additional information on spacing vocabulary such as insets, stacks and inline, refer to Nathan Curtis’s article, [Space in Design Systems](https://medium.com/eightshapes-llc/space-in-design-systems-188bcbae0d62).
+
+{% endcolumn %}
+</div>
+</section>
+
+
+<section class="static-section" markdown="1">
+
+## Containers
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol-12 hxCol-xs-12 hxCol-sm-12 hxCol-md-12 hxCol-lg-12" %}
+
+Pages, cards, modals, panes, etc. are containers  that frame content within a UI. Containers relate directly to the  canvas / grid to create the layout of any page. For this reason, we defined a twenty pixel square inset inside all containers to be harmonious with our column gutters.
+
+{% figure [caption: "Container spacing of twenty pixels (square-inset)."][class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/style/spacing/spacing-system-rule-for-containers.png){: width="893px"}
+{% endfigure %}
+
+{% endcolumn %}
+</div>
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol-12" %}
+
+## Content
+
+Content lives inside the container. Content is stacked vertically using line-heights and bottom margins (depending on type of content). Bottom margins add extra spacing that improves visually perceptible intervals, which is helpful for conveying hierarchy. Content contains:
+
+* Headers are at the highest level of hierarchy for content (page title, section title, sub-section title, container title)
+* Just below the header hierarchy is the data interspersed in the form of paragraphs, lists, forms, and tables that we will refer to as leaf-nodes.
+
+{% endcolumn %}
+
+</div>
+
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+#### Baseline grid is multiples of four
```

Comment:
One more small note; We jump from an H2 Content to three H4's in this section. Perhaps adjust content to address this discrepancy.